packages/scandipwa/src/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -142,7 +142,10 @@ export class MyAccountDispatcher {
             /** @namespace Store/MyAccount/Dispatcher/resetPasswordFetchMutationThen */
             ({ s_resetPassword: { status } }) => dispatch(updateCustomerPasswordResetStatus(status)),
             /** @namespace Store/MyAccount/Dispatcher/resetPasswordFetchMutationError */
-            () => dispatch(updateCustomerPasswordResetStatus('error'))
+            (errors) => {
+                const [{ message }] = errors;
```

Comment:
`errors` is array only if it is graphql error. if promise catches js error, then it will be a regular Error object.
You can use `getErrorMessage` from the `src/util/Request/Error.js`